### ISSUE TITLE
Issues with Lunie and Ledger communicating

### ISSUE THREAD

**Issue by dirtystorm** [luniehq/lunie#2549]
This problem all started yesterday when I tried to delegate atoms.  I delegated the atoms and it flashed a red message as if the transaction did not go through and everything has been going haywire ever since.  E
Everything for my address shows correctly when I check "big dipper", but in Lunie I am getting an error for everything.   

As soon as I log in my balance will bounce back and forth from wrong balance to right balance.  And my ledger no longer communicates properly with Lunie it gives me an error message when trying to withdraw or redeem.   I tried to clear my cache and reset browser, but still having same issues.  Am using chrome browser.  I know that my ledger works and communicates with Lunie as I have deposited and delegated Atoms already, but now it is as if it will not communicate at all...hoping the Atoms are not lost.   

See screenshots below. 
![image](https://user-images.githubusercontent.com/44062905/57296517-cab66d00-7092-11e9-8820-4d49c322f587.png)

![image](https://user-images.githubusercontent.com/44062905/57296580-f20d3a00-7092-11e9-89eb-2624be0f7e09.png)

![image](https://user-images.githubusercontent.com/44062905/57296608-fe919280-7092-11e9-9c6d-8010b2bc5532.png)

**Comment by dirtystorm** [comment]
This is most current screen shots with me logged into my ledger - the balance is wrong here initially and then several seconds later it reverts to the second screen shot which is the correct balance.   


![image](https://user-images.githubusercontent.com/44062905/57296830-97281280-7093-11e9-8c88-d0c69a84bdda.png)

![image](https://user-images.githubusercontent.com/44062905/57296958-e40be900-7093-11e9-8a9a-76d8ba281afb.png)

**Comment by dirtystorm** [comment]
Okay guys I do not know if this will help, but I removed binance chain app from my ledger and now I am able to withdraw rewards - it went through successfully.    But I am still having issues with that 60 Atom delegation...it shows that it is delegated, but not showing up in total  balance.   Now that I think about it this issue may have started after I downloaded the binance chain app and attempted the transaction afterwards.
